### PR TITLE
Prepare repository for npm

### PR DESCRIPTION
**Pull request by ilia-kurganskii**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        with:
+          node-version: 14
+          registry-url: 'https://registry.npmjs.org'
+          cache: 'npm'
+          cache-dependency-path: '**/package-lock.json'
+
+      - uses: actions/cache@v2
+        name: Cache Node Modules
+        with:
+          path: |
```
Comment (diego-antonelli) [inline .github/workflows/release.yml:47]: Pipes are not required for a single line command in the whole file

File: CHANGELOG.md [hunk 2]
```diff
@@ -45,7 +49,7 @@ v3.3.4
  
 v3.3.3
  1. Fix for the db location string [Issue #172] (https://github.com/andpor/react-native-sqlite-storage/issues/172)
- 2. #define in iOS casebase for SQLCIPHER. If you include this #define in your main project settings, this library will pick up appropriate key handling code automatically.
+ 2. define in iOS casebase for SQLCIPHER. If you include this #define in your main project settings, this library will pick up appropriate key handling code automatically.
```
Comment (diego-antonelli) [inline CHANGELOG.md:52]: Is it really necessary to remove the `#`?

File: package.json [hunk 3]
```diff
@@ -23,13 +24,38 @@
     "promise",
     "cordova-sqlite-storage"
   ],
-  "author": "Andrzej Porebski",
+  "files": [
+    "lib/*",
+    "platforms/*"
+  ],
+  "author": "Mendix",
```
Comment (diego-antonelli) [inline package.json:31]: I think we need to keep the original author as well, no?

**Review by diego-antonelli — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (ilia-kurganskii) [reply package.json]: Added

**Review by diego-antonelli — APPROVED** [review]

**Pull request merged**